packages/sanity/src/core/create/studio-app/fetchCreateCompatibleAppId.ts

```diff
@@ -11,51 +10,82 @@ export interface StudioAppResponse {
   appHost: string
 }
 
+export interface StudioApp extends StudioAppResponse {
+  studioUrl: string
+}
+
+export interface CompatibleStudioAppId {
```

Comment:
Could we get a comment providing some high level description what these represents? I.e. it's unclear to me what the difference is between `appId` and `studioApps[].id` - do they represent the same thing?

Reply:
> it's unclear to me what the difference is between appId and studioApps[].id - do they represent the same thing?

They do! As mentioned in an above comment, _studios_ defined in manage are technically named _applications_ , which can definitely be a source of confusion here.

Would it be clearer for you if `appId` was titled `studioAppId`?

Reply:
My confusion was mostly around the fact that there's an CompatibleStudio*AppId* type that has an `appId` property and also a list of studioApps, each with their own id. Am I understanding it correctly that `appId` here reflects which one of the Studio apps in `studioApps` which matches the current origin?

Reply:
Maybe `currentAppId` would be more clear then? Note: consider this as feedback from fresh eyes and not a blocker for merge, I'll leave it to you to judge whether this is something you want to address now or later (or never :))